docs/howTo/requiredSoftware.md

```diff
@@ -0,0 +1,100 @@
+
+</div>
+
+Drag the Android Studio icon into the Applications icon. Double-click the Applications icon, then double-click the Android Studio icon from the applications window.
+
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the Applications window in Finder with the newly installed Android Studio app's icon visible](./assets/requiredSoftware/androidStudioDownload_applicationsWindow.png)
+
+</div>
+
+You may be shown Apple's malicious software dialog; click `Open`
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing Apple's malicious software dialog](./assets/requiredSoftware/androidStudioDownload_appleMaliciousSoftware.png)
+
+</div>
+
+Android Studio should launch to its splash screen, followed by a prompt to install the Android SDK
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the Android Studio Giraffe splash screen](./assets/requiredSoftware/androidStudio_splash.png)
+![Screenshot showing the Android Studio SDK installation prompt](./assets/requiredSoftware/androidStudio_installSDK.png)
+
+</div>
+
+Hit `Next` to continue to the SDK selection screen
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the SDK selection screen](./assets/requiredSoftware/androidStudio_installSDK-selection.png)
+
+</div>
+
+Leave the default selection and click `Next`
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the verify SDK selection screen](./assets/requiredSoftware/androidStudio_verifySelection.png)
+
+</div>
+
+Click `Next` in the Verify Settings screen
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the SDK licence acceptance screen](./assets/requiredSoftware/androidStudio_acceptLicence.png)
+
+</div>
+
+Leave the `android-sdk-license` root node selected, select the `Accept` radio button, then click `Next`.
+
+The download of components will start 
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the SDK components being downloaded](./assets/requiredSoftware/androidStudio_downloadingComponents.png)
+![Screenshot showing the SDK components being downloaded](./assets/requiredSoftware/androidStudio_downloadComponents-finish.png)
+
+</div>
+
+Once the download is complete click `Finish` and you're done!
+
+
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the SDK selection screen](./assets/requiredSoftware/androidStudio_welcome.png)
+
+</div>
+
+Your Android Studio installation and setup is now complete. Next we'll move on to getting the Git repository cloned and opened in the IDE.
```

Comment:
Please can we remove "Next we'll move on to getting the Git repository cloned and opened in the IDE." - per other comment, this leaves like a hanging dependency on a future ticket which is best avoided on this PR. Thanks!